src/capi.cpp

```diff
@@ -1071,6 +1073,9 @@ int rtcAddTrackEx(int pc, const rtcTrackInit *init) {
 			case RTC_CODEC_PCMA:
 				desc.addPCMACodec(init->payloadType);
 				break;
+			case RTC_CODEC_AAC:
+					desc.addAacCodec(init->payloadType, init->latmMuxCfg);
```

Comment:
This will attempt to create a string from a null pointer if `init->latmMuxCfg` is not set.
```suggestion
					desc.addAacCodec(init->payloadType, init->latmMuxCfg ? std::make_optional(string(init->latmMuxCfg)) : nullopt);
```

Reply:
Fixed!